src/components/common/FilterGroup/index.tsx

```diff
@@ -60,9 +60,13 @@ export default function FilterGroup(props: FilterGroupProps): JSX.Element {
   const [filters, setFilters] = useState<Record<string, SearchNodePayload>>(initialFilters);
 
   const onFilterChange = (key: string, filter: SearchNodePayload) => {
-    const newFilt: Record<string, SearchNodePayload> = {};
-    newFilt[key] = filter;
-    setFilters({ ...filters, ...newFilt });
+    if (filter.values.length) {
+      const newFilt: Record<string, SearchNodePayload> = {};
+      newFilt[key] = filter;
+      setFilters({ ...filters, ...newFilt });
```

Comment:
this could just be `setFilter({ ...filters, [key]: filter })`

Reply:
I couldn't figure out the syntax for that. Thanks.